app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to new_user_path
```

Comment:
- [ ] redirect_to edit_user_path if user was saved -- you also should provide the path generator with the user corresponding id. 
    `edit_user_path(@user)`